src/sql/azdata.proposed.d.ts

```diff
@@ -1790,6 +1795,72 @@ declare module 'azdata' {
 		 * The path of the parent node.
 		 */
 		parentNodePath: string;
+		/**
+		 * list of filters that are supported by the node
+		 */
+		filterDefinitions?: NodeInfoFilterDefinition[];
+	}
+
+	export interface NodeInfoFilterDefinition {
+		/**
+		 * The name of the filter property
+		 */
+		name: string;
+		/**
+		 * The type of the filter property
+		 */
+		type: NodeInfoFilterDataType;
+		/**
+		 * The description of the filter property
+		 */
+		description: string;
+	}
+
+	export interface NodeInfoFilterChoiceDefinition extends NodeInfoFilterDefinition {
```

Comment:
not sure what is this for.

btw, please add comments for the new interfaces.

Reply:
Added comment